Sources/PackageDescription4/BuildSettings.swift

```diff
@@ -63,8 +63,8 @@ public struct BuildSettingCondition: Encodable {
     /// At least one parameter is mandatory.
     ///
     /// - Parameters:
-    ///   - platforms: The platforms for which this condition will be applied.
-    ///   - configuration: The build configuration for which this condition will be applied.
+    ///   - platforms: The applicable platforms for this build setting condition.
+    ///   - configuration: The applicable  build configuration for  this build  setting condition.
```

Comment:
nit: there are two spaces between `applicable  build`, `for  this` and `build  setting`